README.md

```diff
@@ -3,9 +3,9 @@
 
 # Budget Module
 
-A budget module is a Cosmos SDK module that implements budget functionality. It is an independent module from other SDK modules and core functionality is to enable anyone to create a budget plan through governance param change proposal. Once it is agreed within the community, voted, and passed, it uses the budget source address to distribute amount of coins by the rate defined in the plan to the collection address. Collecting all budgets and distribution take place every epoch blocks that can be modified by a governance proposal.
+A budget module is a Cosmos SDK module that implements budget functionality. It is an independent module from other SDK modules and core functionality is to enable anyone to create a budget plan through governance param change proposal. Once it is agreed within the community, voted, and passed, it uses the source address to distribute amount of coins by the rate defined in the plan to the destination address. Collecting all budgets and distribution take place every epoch blocks that can be modified by a governance proposal.
```

Comment:
We might as well fix to use `The` to specify the module in this PR as suggested from this https://github.com/tendermint/budget/pull/69#discussion_r738967795

```suggestion
The budget module is a Cosmos SDK module that implements budget functionality. It is an independent module from other SDK modules and core functionality is to enable anyone to create a budget plan through governance param change proposal. Once it is agreed within the community, voted, and passed, it uses the source address to distribute amount of coins by the rate defined in the plan to the destination address. Collecting all budgets and distribution take place every epoch blocks that can be modified by a governance proposal.
```

Reply:
Right, but I think it'll be applied by that PR.

Reply:
I think that PR is just for the review.
https://github.com/tendermint/budget/pull/69#issuecomment-951668573

Reply:
@kogisin Yep, that makes sense. I applied the change!